lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -156,8 +159,13 @@ object NotificationHandler extends CohortHandler {
       currencyISOCode <- requiredField(cohortItem.currency, "CohortItem.currency")
       currencySymbol <- currencyISOtoSymbol(currencyISOCode)
 
-      // In the case of membership price rise, we need to not cap the price
-      cappedEstimatedNewPriceWithCurrencySymbol = s"${currencySymbol}${PriceCap.cappedPrice(oldPrice, estimatedNewPrice, CohortSpec.isMembershipPriceRise(cohortSpec))}"
+      forceEstimated = MigrationType(cohortSpec) match {
+        case Membership2023Monthlies => true
+        case Membership2023Annuals   => true
+        case _                       => false
+      }
```

Comment:
Not sure if this is an improvement or not:
```suggestion
      forceEstimated = MigrationType(cohortSpec) == Membership2023Monthlies ||  MigrationType(cohortSpec) == Membership2023Annuals
```